ci-operator/config/openstack-k8s-operators/watcher-operator/openstack-k8s-operators-watcher-operator-main.yaml

```diff
@@ -0,0 +1,40 @@
+base_images:
+  cli:
+    name: "4.16"
+    namespace: ocp
+    tag: cli
+  opm-builder:
+    name: opm-builder
+    namespace: ci
+    tag: latest
+build_root:
+  from_repository: true
+images:
+- dockerfile_literal: |
+    FROM src
+    RUN pip3 install pre-commit
+  from: src
+  to: openstack-operator-ci-pre-commit
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: precommit-check
+  commands: |
+    export SKIP=no-commit-to-branch
+    export GOFLAGS=
+    export PRE_COMMIT_HOME=/tmp
+    export GOLANGCI_LINT_CACHE=/tmp
+    mkdir -p ../operator/ && cp -r . ../operator
+    cd ../operator
+    pre-commit run -a --show-diff-on-failure
```

Comment:
ok so this cant pass until we merge https://github.com/openstack-k8s-operators/watcher-operator/pull/4

so ill manually checkout that pr and run pre-commit and confirm it passes.
if it does we can merge that pr then proceed with this one.

Reply:
(that merged now and job is green here, adding lgtm label)